sourced/ml/algorithms/id_splitter/features.py

```diff
@@ -0,0 +1,125 @@
+import logging
+import string
+import tarfile
+import warnings
+
+import numpy as np
```

Comment:
Can you please check how do we normally import numpy in this project. If it is `numpy` in most of the places, should be changed